org.osgi.test.common/src/test/java/org/osgi/test/common/dictionary/DictionariesTestCase.java

```diff
@@ -225,6 +276,7 @@ public void null_checks() {
 		assertThat(asMap.remove(null)).isNull();
 		assertThat(asDictionary.get(null)).isNull();
 		assertThat(asDictionary.remove(null)).isNull();
+		assertThat(asDictionarySr.get(null)).isNull();
```

Comment:
When doing multiple assertions in a test method, it is usually helpful to use the `as()` method of AssertJ to describe the assertion. That way when it fails you've got more of an idea what's going on rather than having to dig through the source (and sometimes the line numbers change when you edit). Eg:

```java
assertThat(asMap.remove(null)).as("asMap.remove").isNull();
assertThat(asDictionary.get(null)).as("asDictionary.get").isNull();
```

It can be a bit laborious though so in simple tests I am usually a bit more lenient with this rule. Just letting you know.